core/stores/cache/cache_test.go

```diff
@@ -45,7 +49,7 @@ func (mc *mockedNode) Get(key string, v interface{}) error {
 }
 
 func (mc *mockedNode) IsNotFound(err error) bool {
-	return err == mc.errNotFound
+	return errors.Is(err, mc.errNotFound)
```

Comment:
Changed semantics, this modification requires discussion.

Reply:
no problem.